packages/webapp/src/utils/credits-estimation-constants.js

```diff
@@ -0,0 +1,21 @@
+const CREDITS_ESTIMATION_PATH = '/credits-estimation'
+
+const creditEstimateRoutes = {
+  ESTIMATOR_CREDITS_COST: '/credits-estimation/credits-cost',
+  ESTIMATOR_CREDITS_TIER: '/credits-estimation/credits-tier'
+}
+
+const views = Object.fromEntries(
+  Object.entries(creditEstimateRoutes).map(([k, v]) => [k, v.substring(1)])
+)
+
+const redisKeys = {
+  ESTIMATOR_CREDITS_CALCULATION: 'estimator-credits-calculation'
+}
+
+export default {
+  CREDITS_ESTIMATION_PATH,
+  creditEstimateRoutes,
```

Comment:
Please replace this with routes following the split into distinct files for credits estimation and credits purchase.

Reply:
done